pkg/controller/kibana/controller.go

```diff
@@ -60,8 +62,8 @@ var (
 )
 
 func (r *ReconcileKibana) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-
-	if err := kibana.ReconcileKibanaInstance(request, r.client); err != nil {
+	esClient := elasticsearch.NewClient(constants.ElasticsearchDefaultImage, request.Namespace, r.client)
```

Comment:
This seems odd to me.  why does the client need the image?

Reply:
This is a mistake. This supposed to be the name of the Elasticsearch custom resource.